charts/sscs-evidence-share/values.yaml

```diff
@@ -3,6 +3,7 @@ java:
   applicationPort: 8091
   readinessPath: '/health/readiness'
   aadIdentityName: sscs
+  devmemoryLimits: 2Gi
```

Comment:
is there a reason we are now setting this limit?

Reply:
I couldn't deploy to preview without it